### PR TITLE
Implement Invalid opcode

### PR DESCRIPTION
**Pull request by MarcosNicolau**
The spec doesn't mention how to handle the invalid opcode. The vm2 stops the execution altogether. In our case, we return an Err and let the `handleError` run.

### CODE REVIEW

**Comment by MarcosNicolau** [comment]
Closes #131

**Comment by JulianVentura** [comment]
LGTM

**Review by jorbush — APPROVED** [review]
LGTM

**Review by JulianVentura — APPROVED** [review]

File: src/lib.rs [hunk 1]
```diff
@@ -150,7 +150,7 @@ pub fn run(
 
         if vm.predicate_holds(&opcode.predicate) {
             let result = match opcode.variant {
-                Variant::Invalid(_) => todo!(),
+                Variant::Invalid(_) => Err(EraVmError::OpcodeError(OpcodeError::InvalidOpCode)),
```
Comment (gianbelinche) [inline src/lib.rs:155]: nit: you could do `OpcodeError::InvalidOpCode.into()` since `EraVmError` implements from for it

**Review by gianbelinche — APPROVED** [review]

**Pull request merged**